### PR TITLE
Disable and clear connection metadata cache before baking model

### PR DESCRIPTION
**Pull request by othercorey**
Closes https://github.com/cakephp/bake/issues/724

### CODE REVIEW

**Comment by othercorey** [comment]
@ADmad Do you think this will clear the cache properly?

**Comment by ADmad** [comment]
Disabling schema cache will probably make it just not check it (which is fine as far as bake goes), but it won't clear it. When using the generated model it would most likely still use the stale schema cache.

**Comment by othercorey** [comment]
> Disabling schema cache will probably make it just not check it (which is fine as far as bake goes), but it won't clear it. When using the generated model it would most likely still use the stale schema cache.

Right, sorry. I meant will this avoid the cache problem. I was thinking the user would only bake locally, but they're probably testing on the same instance.

File: src/Command/ModelCommand.php [hunk 1]
```diff
@@ -78,6 +80,16 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             return static::CODE_SUCCESS;
         }
 
+        // Disable caching before baking with connection
+        $connection = ConnectionManager::get($this->connection);
+        if ($connection instanceof Connection) {
+            $collection = $connection->getSchemaCollection();
+            if (!$collection instanceof SubsetSchemaCollection) {
+                $connection->cacheMetadata(false);
+                $connection->getCacher()->clear();
```
Comment (othercorey) [inline src/Command/ModelCommand.php:89]: Added call to clear cache explicitly.

**Review by markstory — COMMENTED** [review]
This looks like it would work to me.

**Comment by Theaxiom** [comment]
Thank you! 👍

**Pull request merged**